lib/app/components/pages/profile/views/profile_view.dart

```diff
@@ -8,16 +8,18 @@ class ProfileView extends GetView<ProfileController> {
   const ProfileView({Key? key}) : super(key: key);
   @override
   Widget build(BuildContext context) {
+    // todo 仮としてユーザ名を取得。本来はIDが必要
+    controller.userName(Get.arguments[0]);
     return Scaffold(
       appBar: AppBar(
         title: const Text('ProfileView'),
         centerTitle: true,
       ),
-      body: const Center(
-        child: Text(
-          'ProfileView is working',
-          style: TextStyle(fontSize: 20),
-        ),
+      body: Center(
+        child: Obx(() => Text(
+              '${controller.userName.value}\'s ProfileView is working',
+              style: const TextStyle(fontSize: 20),
+            )),
```

Comment:
```suggestion
        child: Text(
              '${controller.userName}\'s ProfileView is working',
              style: const TextStyle(fontSize: 20),
            ),
```
④こちらはRxString型じゃなくなるので`Obx`がいらなくなります。